package.json

```diff
@@ -31,9 +31,11 @@
     "bootstrap": "^4.6.0",
     "core-js": "^3.9.1",
     "d3": "^6.7.0",
+    "file-saver": "^2.0.5",
     "hammerjs": "^2.0.8",
     "jquery": "^3.6.0",
     "mobile-device-detect": "^0.4.3",
+    "moment": "^2.29.1",
```

Comment:
I see that we need this only to help format one date at a specific place. Please find another than to import `moment.js` to the whole project, momentjs is not yet a ES6 module ready project, and does not support tree shaking. Meaning the whole lib will be included in our deliverable JS, without removing what we don't use, resulting in very very big JS files for not much use.

Maybe a simple `new Date().toISOString()` could suffice for what you want to accomplish